packages/wrangler/package.json

```diff
@@ -7,6 +7,7 @@
 		"wrangler": "./bin/wrangler.js",
 		"wrangler2": "./bin/wrangler.js"
 	},
+	"main": "wrangler-dist/cli.js",
```

Comment:
Rather than conflating the two, I wonder if we should actually have two generated entry-points for wrangler?

```
wrangler-dist/index.js
wrangler-dist/cli.js
```

The first would only contain the `unstable_dev()` function (for now).
The second would only contain the binary that is run when Wrangler is used as a CLI.

Then `"main"` would just point at the index.js.

Reply:
👍  all my comments can be follow-ups

Reply:
Does sound like additional scope to me.